### PR TITLE
Function Docs and PR Template

### LINKED ISSUE #22: Function Docs
**Describe the bug**
A lot of functions in GSW are missing documentation which would be nice for godoc to autogenerate web pages of our code

### PR DESCRIPTION
**Pull request by AarC10**
This closes #22

# Description
Add a PR template and documentation for each function in GSW. Also CI will autoformat code now.

# Checklist:
- [x] New functionality is documented in the necessary spots (i.e new functions documented in the header)
- [x] Unit tests cover any new functionality or edge cases that the PR was meant to resolve (if applicable) 
- [x] The CI checks are passing
- [x] I reviewed my own code in the GitHub diff and am sure that each change is intentional

### CODE REVIEW

File: data/config/backplane.yaml [hunk 1]
```diff
@@ -227,7 +227,7 @@ measurements:
       endianness: little
 telemetry_packets:
   - name: PowerModule
-    port: 11000
+    port: 11015
```
Comment (apenza) [inline data/config/backplane.yaml:230]: why?

**Review by apenza — APPROVED** [review]
Nice docs and template
